Formula/ignition-garden.rb

```diff
@@ -47,7 +47,7 @@ def install
   end
 
   test do
-    yaml_file = share/"ignition/ignition-garden/gazebodistro/collection-garden.yaml"
+    yaml_file = share/"gz/gz-garden/gazebodistro/collection-garden.yaml"
```

Comment:
I think these install paths are not updated yet

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_garden-install_bottle-homebrew-amd64&build=233)](https://build.osrfoundation.org/job/ignition_garden-install_bottle-homebrew-amd64/233/) https://build.osrfoundation.org/job/ignition_garden-install_bottle-homebrew-amd64/233/

~~~
vcs validate: error: argument --input: can't open
 '/usr/local/Cellar/ignition-garden/0.999.999~0~20220414/share/gz/gz-garden/gazebodistro/collection-garden.yaml':
 [Errno 2] No such file or directory:
 '/usr/local/Cellar/ignition-garden/0.999.999~0~20220414/share/gz/gz-garden/gazebodistro/collection-garden.yaml'
~~~

Reply:
Yeah we haven't migrated `IGN_DATA_INSTALL_DIR` yet. This was future-proofing. I understood no bottles were being built yet.

Reply:
the garden formula is a special case; we have a nightly job to build that formula and run its test. I had forgotten about this when saying that tests were only used when building bottles. I'll submit a small PR to fix this formula only

Reply:
https://github.com/osrf/homebrew-simulation/pull/1948